src/ExpenseTracker.Web/Views/Workspace/Edit.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@model ExpenseTracker.Web.ViewModels.Workspace.WorkspaceEditViewModel
+
+@{
+    ViewBag.Title = "Edit";
```

Comment:
@BirajMainali I think we need a more descriptive title. `Edit workspace` for instance might be better.